pages/windows/dir.md

```diff
@@ -17,3 +17,7 @@
 - Show the contents of a given directory, including hidden ones:
 
 `dir {{path/to/directory}} /A`
+
+- Show the contents of the current directory and all subdirectories:
```

Comment:
No need to repeat "Show the contents of the current directory" after the first time. Just mention - "Show the contents of subdirectories too"

Reply:
Hello @agnivade, thank you for your feedback;
resubmitted, please be so kind and close this PR - sorry for the inconvenience 
Thank you,
regards, Ben